test/vso_tools/build_vm/change_ssh_port.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Suggest adding a readme in build_vm so future maintainers will know when should they run this.